maintainers/maintainer-list.nix

```diff
@@ -4825,6 +4825,12 @@
     githubId = 55911173;
     name = "Gwendolyn Quasebarth";
   };
+  larusso = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add larusso``.